### ISSUE TITLE
Features Grid Updates Process documentation

### ISSUE THREAD

**Issue by vincent6767** [o3de/o3de#12759]
Features grid is part of the documentation we need to update on the release. This issue tracks the effort to document the process, so everyone understands
1. What's the features grid?
2. What's the purpose of the features grid?
3. Where can I find it? 
4. How to update it?
5. The ideal timeline for updating the features grid.

**Comment by vincent6767** [comment]
Documentation aimed at contributors.

1. What's the features grid?
	1. A feature grid to allow members to understand the maturity and state of our documented and undocumented features. Currently being maintained by each SIG and published by the SIG docs community.
2. What's the purpose of the features grid?
	1. Quoted from [the original RFC](https://github.com/o3de/o3de/issues/6821)
		1. The idea is so that a studio, company, developer, or member can look at this grid, and understand the state of the modules that are important to them. This also helps serve as a guide to where we need to focus our efforts and not duplicate work. The initially proposed tables are for Feature maturity, Code maturity, and Binary Maturity.
3. Where can I find it?
	1. You can find the features grid in each release: [Release Notes for Open 3D Engine - Open 3D Engine (o3de.org)](https://www.o3de.org/docs/release-notes/)
	2. Here is the repository for the feature grid: [community/features at main · o3de/community · GitHub](https://github.com/o3de/community/tree/main/features)
4. How to update it?
	1. You can see the instruction here: [community/features at main · o3de/community · GitHub](https://github.com/o3de/community/tree/main/features#update-form-here-feature-state-form) .
5. What's the suggested process to update the O3DE release?
	1. The release manager creates issues in the release board, capturing the tasks for each SIG to update the features grid.
		1. The features grid should be completed at max one week before the release day.
	2. Each SIG updates the features grid and sends a PR, requesting approval from at least one SIG docs community member and their own SIG members.
	3. SIG member merges the PR.
	4. Once the features grid is completed, the release manager notifies the SIG Docs community to prepare for the feature grid updates.
	5. SIG docs community merged the features grid updates together with other documentation changes.

**Comment by vincent6767** [comment]
@tonybalandiuk  Please review. If it looks good to you, we can have this added to your release documentation

**Comment by FiniteStateGit** [comment]
SIG-Docs-Community has an RFC in motion re: the feature grid as well: https://github.com/o3de/sig-docs-community/issues/64.

**Comment by vincent6767** [comment]
@FiniteStateGit Yep, saw that. But that will be more of the content as far as I understand. This issue tracks the process updating the features grid near the release.

**Comment by vincent6767** [comment]
Since these is no traction for this issue, I'll close it. Thank you guys!
